src/bench.ts

```diff
@@ -207,6 +207,22 @@ export class Bench extends EventTarget {
     return values
   }
 
+  runSync (): Task[] {
+    if (this.concurrency != null) {
+      throw new Error('Cannot use `concurrency` option when using `runSync`')
+    }
```

Comment:
Please do the same for the `runSync` method in the `Task` class and we're good to go (yes, it's bad to have `Task` export not only as a type but unfortunately the API have already been widely abused)